packages/connectivity/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 0.4.4+1
 
 * Update and migrate iOS example project.
+* Define clang module for iOS.
```

Comment:
Need a new version for this commit.

Reply:
Even though the published one is 0.4.4?

Reply:
Oops. Then it is ok to just add to the same version I guess :)

Reply:
I defer to you, I can add a new version if that's what you prefer.